server.js

```diff
@@ -16,6 +17,19 @@ app.use(
   express.static(path.join(__dirname, "client/storybook-static"))
 );
 
+app.get("/api/survey/:id", async (req, res) => {
```

Comment:
In RESTful APIs, you would use plural here

```suggestion
app.get("/api/survies/:id", async (req, res) => {
```

Reply:
Is `survey` equal to `question`?
If so, consider reducing it to a single name.